app/forms/support_form.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class SupportForm
+  include ActiveModel::Model
+
+  attr_accessor :message, :participant_profile_id, :school_id, :current_user
+  attr_reader :subject
+
+  validates :message, presence: { message: "Please provide details of your request" }
```

Comment:
We generally don’t use 'please' in error messages 😄 

```suggestion
  validates :message, presence: { message: "Enter your message" }
```